.github/workflows/test.yml

```diff
@@ -82,6 +82,7 @@ jobs:
             --set=controllerManager.manager.image.version=${{ inputs.K8S_OPERATOR_IMAGE_TAG }}
           sleep 5
           kubectl wait --for=condition=Ready -n default --all pods
+          sleep 30
```

Comment:
why sleep 30 after blocking on `kubectl wait --for=condition=Ready -n default --all pods`?

Reply:
There's a very long thread about it, but SSL cert provisioning is currently not finished until well after the pod marks itself as ready. The operator team is looking into fixing that.